cicd-scripts/run-e2e-tests.sh

```
@@ -48,6 +48,13 @@ else
   kubectl apply -f ${ROOTDIR}/operators/multiclusterobservability/config/crd/bases --server-side=true --force-conflicts
 fi
 
+# apply network policies in case they changed
+# to make sure we do not cause breakage
+if [[ -z ${IS_KIND_ENV} ]]; then
+  oc create namespace open-cluster-management-observability
+  oc apply -f ${ROOTDIR}/operators/multiclusterobservability/config/networkpolicies
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)